sunpycube/cube/cube_utils.py

```diff
@@ -267,19 +267,19 @@ def getitem_3d(cube, item):
         The item to get from the cube
     """
     axes = cube.axes_wcs.wcs.ctype
-    slice_to_map = (axes[-2] != 'WAVE' and
+    slice_to_map = (axes[1] != 'WAVE' and
```

Comment:
the change in orientation.
before [space, space, wave, time] 
after    [time, wave, space, space]